genon/preprocessor/converters/hwp_to_pdf/libreoffice.py

```
@@ -0,0 +1,82 @@
+                shutil.copy2(in_path, ascii_copy)
+                candidates = [ascii_copy, in_path]
+
+            timeout = int(os.environ.get("HWP_TO_PDF_TIMEOUT_SEC", DEFAULT_TIMEOUT_SEC))
+            for cand in candidates:
+                cmd = [
+                    "soffice", "--headless",
+                    "--convert-to", convert_arg,
+                    "--outdir", str(out_dir),
+                    str(cand),
+                ]
+                proc = subprocess.run(cmd, env=env, capture_output=True, text=True, timeout=timeout)
```

Ruff (0.15.13)

[error] `subprocess` call: check for execution of untrusted input

(S603)